carbon-feature-plugin/src/main/java/org/wso2/maven/p2/feature/install/FeatureInstallMojo.java

```diff
@@ -111,6 +130,18 @@ private FeatureInstaller constructFeatureInstaller() {
         resourceBundle.setLauncher(this.launcher);
         resourceBundle.setForkedProcessTimeoutInSeconds(this.forkedProcessTimeoutInSeconds);
         resourceBundle.setLog(getLog());
+        resourceBundle.setRuntimeLocation(this.runtimeLocation);
         return new FeatureInstaller(resourceBundle);
     }
+
+    private DirectorRuntime getDirectorRuntime() throws MojoFailureException, MojoExecutionException {
+        switch (directorRuntime) {
```

Comment:
In tycho, it supports two runtimes: standalone and embedded osgi runtime. Currently we support only standalone runtime. Added this switch case so that we can introduce the embedded osgi runtime later.